### PR TITLE
Placeholder about page with button in the drink list header

### PR DESCRIPTION
**Pull request by audiodude**
screenshot 2016-06-20 21 04 15

screenshot 2016-06-20 21 10 46

### CODE REVIEW

**Comment by audiodude** [comment]
I messed up the order rating screen, need to fix.

**Comment by audiodude** [comment]
Okay I fixed it with a hack. I feel very ashamed of this hack so I guess....welcome to nebree8 development?

**Comment by funrollloops** [comment]
That is a hack, isn't it?

We could change it so that the order-status element actually takes up space on the screen instead of being absolutely positioned. The fixed positionining was just a quick hack to get it working.

This works fine for now though.

File: static/components/order-status/order-status.ts [hunk 1]
```diff
@@ -38,9 +38,11 @@ class OrderStatusCtrl {
 
 class OrderStatusService {
   orders: Order[] = [];
+  showing: boolean;
```
Comment (funrollloops) [inline static/components/order-status/order-status.ts:41]: You could initialize showing inline here, i.e.

showing: boolean = true;

File: static/drink-list/drink-list.ts [hunk 2]
```diff
@@ -3,13 +3,19 @@ class DrinkListCtrl {
...
 
-  constructor(private $scope: angular.IScope,
+  constructor(private OrderStatusService: OrderStatusService,
+              private $scope: angular.IScope,
               private $location: angular.ILocationService,
               private $timeout: angular.ITimeoutService,
               private $window: angular.IWindowService,
               DrinkListStateService: DrinkListStateService,
               DrinksService: DrinksService) {
+    this.orderStatusService = OrderStatusService;
```
Comment (funrollloops) [inline static/drink-list/drink-list.ts:15]: OrderStatusService is already being copied to a local variable with the same name as the argument; that's what the "private" means in the constructor parameter list above.

File: static/app.ts [hunk 3]
```diff
@@ -1,3 +1,15 @@
+class TopCtrl {
+  orderStatusService: OrderStatusService;
+
+  constructor(private OrderStatusService: OrderStatusService) {
```
Comment (funrollloops) [inline static/app.ts:4]: ditto here; if you want to give the member variable a different name, you can drop "private" from the argument list.

**Comment by audiodude** [comment]
I think the issue is that the `<order-status>` isn't part of ng-view and thus can't refer to the view's controller, otherwise we could just do `AboutCtrl.showOrderStatus = false`. Not the fixed positioning. Though I guess if it had static positioning, it would just fall off the bottom and not get displayed and there would be no need to hide it. I thought the fixed positioning had to do with the animation and keeping it at the bottom as users scroll.

**Comment by funrollloops** [comment]
Having a TopCtrl makes sense, though having the about screen hide the order
status by wiring a bool through three controllers is definitely a little
hacky.

**Pull request merged**